reactor-core/src/main/java/reactor/core/Scannable.java

```diff
@@ -211,7 +208,7 @@ public interface Scannable {
 		public static final Attr<Boolean> TERMINATED = new Attr<>(false);
 
 		/**
-		 * A {@link Stream} of {@link reactor.util.function.Tuple2} representing key/value
+		 * A {@link Stream} of {@link Tuple2} representing key/value
```

Comment:
Is short link allowed or full definition is required according to the code style?